### PR TITLE
add doc to typed-node-list-p

### PR DESCRIPTION
**Pull request by Jobhdez**

### CODE REVIEW

File: src/typechecker/typed-node.lisp [hunk 1]
```diff
@@ -9,6 +9,7 @@
   (unparsed (required 'unparsed) :type t         :read-only t))
 
 (defun typed-node-list-p (x)
+  "Returns True if every element in the list X is of TYPED-NODE."
```
Comment (stylewarning) [inline src/typechecker/typed-node.lisp:12]: "Is every element of the list X a TYPED-NODE?"

**Review by stylewarning — CHANGES_REQUESTED** [review]

**Comment by eliaslfox** [comment]
@stylewarning or @gefjon feel free to weight in. But personally I don't see the point in adding docstrings to these functions. Especially because they only exist for the subsequent deftype.

**Comment by stylewarning** [comment]
> @stylewarning or @gefjon feel free to weight in. But personally I don't see the point in adding docstrings to these functions. Especially because they only exist for the subsequent deftype.

In this case, for the reason you state, I'm ambivalent. You're right that these kinds of functions aren't intended to really be used directly.

**Comment by eliaslfox** [comment]
Closing this for reasons described above. If the type checker used per file packages, then this function wouldn't be exported.

**Pull request closed without merging**